index.html

```diff
@@ -264,8 +279,8 @@ <h2>
           </li>
         </ul>
       </div>
-      <p class="email-address">mihreteabdemeke1@gmail.com</p>
-    </section>
+      <p class="only-mobile email-address">mihreteabdemeke1@gmail.com</p>
+    </footer>
```

Comment:
- [ ] _[OPTIONAL]_ there is quite a white space after your footer, and your `footer` is inside your `main` tag. please make sure to move it and fix the white space.